.github/wiki/Contribution.md

```diff
@@ -0,0 +1,238 @@
+# Adding a new addon
+
+There are 3 types of addons.
+1. API
+2. Live
+3. Web
```

Comment:
we have Mix addon as well Top :) 
![image](https://user-images.githubusercontent.com/11751745/157224453-dd3631d2-d576-471c-8f3c-8e2db2c9be91.png)